templates/index.twig

```diff
@@ -3,13 +3,24 @@
 {% block content %}
 <div class="col-md-12">
 	{% include "article-list.twig" with {"wordCount": 100} %}
-	<div class="pagination-buttons">
+<div class="text-center">
+	<ul class="pagination pagination-centered">
 		{% if pagination.prev %}
-		<a href="{{pagination.prev.link}}" class="prev {{pagination.prev.link|length ? '' : 'invisible'}} pull-left"><button class="btn btn-default">Previous</button></a>
+		<a href="{{pagination.prev.link}}" class="prev {{pagination.prev.link|length ? '' : 'invisible'}}"><button class="btn btn-default">Previous</button></a>
 		{% endif %}
+		{% for page in pagination.pages %}
+				<li>
+						{% if page.link %}
+								<a href="{{page.link}}" class="btn btn-default">{{page.title}}</a>
+						{% else %}
+								<span class="btn btn-default">{{page.title}}</span>
+						{% endif %}
+				</li>
+		{% endfor %}
 		{% if pagination.next %}
-		<a href="{{pagination.next.link}}" class="next {{pagination.next.link|length ? '' : 'invisible'}} pull-right"><button class="btn btn-default">Next</button></a>
+		<a href="{{pagination.next.link}}" class="next {{pagination.next.link|length ? '' : 'invisible'}}"><button class="btn btn-default">Next</button></a>
```

Comment:
Also, you shouldn't have a `<button>` inside the `<a>`. You can just put "Next" directly inside the `<a>`